lib/Doctrine/Import/Mysql.php

```diff
@@ -31,6 +31,12 @@
  */
 class Doctrine_Import_Mysql extends Doctrine_Import
 {
+    /**
+     * @var Doctrine_Connection_Mysql $conn       Doctrine_Connection object, every connection
+     *                                            module holds an instance of Doctrine_Connection
+     */
+    protected $conn;
```

Comment:
Same as the change like this above (to change the docblock's class on the property).